pkg/context/context.go

```diff
@@ -25,7 +25,7 @@ import (
 type Context interface {
 	Next()
 	Abort()
-
+	AbortAndLog(string, error)
```

Comment:
I don't think add a method for this interface is a good idea.

Reply:
renamed